scripts/unpack_ddf.py

```diff
@@ -265,18 +266,27 @@ def print_shader_code(text):
 
 
 PRINTERS = {}
-PRINTERS['.luac']       = print_lua_file
-PRINTERS['.vpc']        = print_shader_file
-PRINTERS['.fpc']        = print_shader_file
-PRINTERS['.cpc']        = print_shader_file
+PRINTERS['luac']       = print_lua_file
+PRINTERS['vpc']        = print_shader_file
+PRINTERS['fpc']        = print_shader_file
+PRINTERS['cpc']        = print_shader_file
 
 if __name__ == "__main__":
     path = sys.argv[1]
 
     with open(path, 'rb') as f:
         content = f.read()
-        _, ext = os.path.splitext(path)
-        builder = BUILDERS.get(ext, None)
+        ext = path.split(".")
+        if ext[-1] == "lz4":
```

Comment:
What's the reasoning for not doing `os.splitext()` here?
When are we not having a suffix?